hw/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import './Input.css'
+
+type InputProps = {
+    value: string;
+    placeholder: string;
+    onChange: (e:any) => void
```

Comment:
в callback должно сразу попадать `string`
```suggestion
    onChange: (value: string) => void
```